modules/imgproc/src/accum.simd.hpp

```diff
@@ -0,0 +1,2740 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
```

Comment:
Please add `#include "opencv2/core/hal/intrin.hpp"` here.

Warning comes from these files:
> In file included from /build/precommit_macosx/build/modules/imgproc/accum.sse2.cpp

BTW, This file is autogenerated by CMake (`.sse2.cpp`, `.avx.cpp` and `.neon.cpp`) in build directory. This may confuse because you have files with similar names in "src" directory.